nfc/CHANGELOG.rst

```diff
@@ -9,6 +9,21 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+master
+******
+
+Added
+=====
+* Added dedicated header file ``nrf_nfc_errno.h`` with error codes.
+* Added source file which translates NRFX error codes to NFC library error codes.
+* Added specification for returned error codes in an API documentation.
+* Removed unit tests dependencies from header files ``nfc_t2t_lib.h`` and ``nrf_t4t_lib.h``.
+
+Modified
+========
+
+* Specific errors codes from ``nrf_nfc_errno.h`` are returned by library instead of Zephyr error codes.
```

Comment:
```suggestion

Added
=====

* A dedicated header file :file:`nrf_nfc_errno.h` with error codes.
* A source file that translates nrfx error codes to NFC library error codes.
* Definitions for returned error codes in the API documentation.

Modified
========

* The library returns specific errors codes from the :file:`nrf_nfc_errno.h` file instead of Zephyr error codes.
* Removed unit tests dependencies from the header files :file:`nfc_t2t_lib.h` and :file:`nrf_t4t_lib.h`.

```

Reply:
Thanks. Done